CMakeLists.txt

```diff
@@ -66,39 +66,44 @@ set(BUILD_SHARED_LIBS ON CACHE BOOL "Build shared")
 # Include cmake scripts
 include(cmake/Dependencies.cmake)
 
-# Detect compiler support for target ID
-# This section is deprecated. Please use rocm_check_target_ids for future use.
-if( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" )
-    execute_process(COMMAND ${CMAKE_CXX_COMPILER} "--help"
-        OUTPUT_VARIABLE CXX_OUTPUT
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-        ERROR_STRIP_TRAILING_WHITESPACE)
-    string(REGEX MATCH ".mcode\-object\-version" TARGET_ID_SUPPORT ${CXX_OUTPUT})
-endif()
+option(ROCRAND_CMAKE_HIP_LANGUAGE "Use CMake's HIP language support" OFF)
+if(ROCRAND_CMAKE_HIP_LANGUAGE)
```

Comment:
That's an interesting thought. I'll change this behaviour from `ROCRAND_CMAKE_HIP_LANGUAGE=ON` to `ROCRAND_CMAKE_LANG=HIP`. The default would be `ROCRAND_CMAKE_LANG=CXX`, and if somebody wants to implement CMake CUDA language support, it would be `ROCRAND_CMAKE_LANG=CUDA`.